### PR TITLE
[security] Add details about vendor announce list

### PR DESCRIPTION
**Pull request by gouthamve**
This is an RFC and no finalised. Inviting all the opinions! This is based on https://github.com/kubernetes/security/blob/master/private-distributors-list.md

### CODE REVIEW

**Comment by gouthamve** [comment]
Open questions: 
1. Should we have a formal maintainer vote for admission to this list? (Seems overkill, lazy consensus would be fine)
2. Should the members of this list be public or not? (Public makes sense, k8s does public)

**Comment by tomwilkie** [comment]
> Should we have a formal maintainer vote for admission to this list? (Seems overkill, lazy consensus would be fine)

I'd be happy with "lazy consensus, requires two maintainers as per a PR".  I think we should avoid maintainers being able to unilaterally adding or removing people to the list.

> Should the members of this list be public or not? (Public makes sense, k8s does public)

If we make it public and require a PR, that also solved the above problem - so I'd vote yet.

**Review by tomwilkie — APPROVED** [review]

**Review by pracucci — APPROVED** [review]
LGTM

**Review by pstibrany — APPROVED** [review]
LGTM

File: SECURITY.md [hunk 1]
```diff
@@ -5,3 +5,77 @@ As with any complex system, it is certain that bugs will be found, some of them
...
+
+### Membership Criteria
+
+To be eligible for the cortex-vendors-announce@googlegroups.com mailing list, your
+company should:
+
+0. Have an actively monitored security email alias for our project.
+1. Have a public hosted version/distribution of Cortex.
+2. Have a user base not limited to your own organization.
+4. Not be a downstream or rebuild of another vendor.
```
Comment (ranton256) [inline SECURITY.md:69]: 3?

Comment (pracucci) [inline SECURITY.md:69]: Not sure this is much clear. Could you clarify what you mean?

File: SECURITY.md [hunk 2]
```diff
@@ -5,3 +5,77 @@ As with any complex system, it is certain that bugs will be found, some of them
...
+Group membership is managed by the Cortex maintainers using Google Groups.
+
+### Membership Criteria
+
+To be eligible for the cortex-vendors-announce@googlegroups.com mailing list, your
+company should:
+
+0. Have an actively monitored security email alias for our project.
+1. Have a public hosted version/distribution of Cortex.
+2. Have a user base not limited to your own organization.
```
Comment (pracucci) [inline SECURITY.md:68]: Item (3) is missing.

File: SECURITY.md [hunk 3]
```diff
@@ -5,3 +5,77 @@ As with any complex system, it is certain that bugs will be found, some of them
...
 The public disclosure date is agreed between the Cortex Team and the bug submitter. We prefer to fully disclose the bug as soon as possible, but only after a mitigation or fix is available. We will ask for delay if the bug or the fix is not yet fully understood or the solution is not tested to our standards yet. While there is no fixed timeframe for fix & disclosure, we will try our best to be quick and do not expect to need the usual 90 days most companies ask or. For a vulnerability with a straightforward mitigation, we expect report date to disclosure date to be on the order of 7 days.
+
+---------------------------
+
+## Private Vendors List
+
+Cortex is being used by several vendors to provide a hosted Prometheus experience
+to their users. We also have list that is used to provide actionable information
+to multiple vendors at once. This list is not intended for individuals to find out about
+security issues.
```
Comment (bboreham) [inline SECURITY.md:16]: I find this wording a bit confusing.  How about replacing the whole paragraph with
"We also have a list for vendors who provide Cortex to their users."

**Review by bboreham — APPROVED** [review]
Thanks

**Pull request merged**